NodeTypes/Form.fusion

```diff
@@ -8,21 +8,27 @@ prototype(Sitegeist.PaperTiger:Form) < prototype(Neos.Neos:ContentComponent) {
     renderer = Neos.Fusion:Case {
         runtimeForm {
             condition = ${node.context.live}
-            renderer = Neos.Fusion.Form:Runtime.RuntimeForm {
-                @cache.context.formClass = '_formClass'
-                namespace = ${q(node).property('_identifier')}
-                attributes.class = ${_formClass}
-                process = Neos.Fusion.Form:Runtime.SingleStepProcess {
-                    content = Sitegeist.PaperTiger:FieldCollection.Renderer {
-                        nodePath = 'fields'
+
+            renderer = Neos.Fusion:Join {
+                anchor = afx`<a id={'form_' + node.identifier}></a>`
```

Comment:
Shouldn't it be enough to give the form tag the id? I don't like empty anchor that much

Reply:
Not after finishing as then no form is rendered

Reply:
Hm, ok. But in that case, it should be a prop, so that it is possible to change it (class, aria-labels, tag name, etc)

Reply:
#23